extensions/amp-brightcove/0.1/amp-brightcove.js

```diff
@@ -121,7 +121,7 @@ class AmpBrightcove extends AMP.BaseElement {
    *
    * See https://github.com/ampproject/amphtml/issues/2224 for information.
    */
-  unlayout() {
+  unlayoutCallback() {
```

Comment:
@mkhatib This should have also `@override` jsdoc annotation. Please separate this into a separate PR and merge ASAP - no need to go through review.